common/src/test/scala/com/gu/zuora/ZuoraSpec.scala

```diff
@@ -12,17 +13,45 @@ import scala.concurrent.duration._
 @IntegrationTest
 class ZuoraSpec extends AsyncFlatSpec with Matchers with LazyLogging {
 
+  def uatService: ZuoraService = new ZuoraService(zuoraConfigProvider.get(true), RequestRunners.configurableFutureRunner(30.seconds))
+
   "ZuoraService" should "retrieve an account" in {
-    val zuoraService = new ZuoraService(Configuration.zuoraConfigProvider.get(), RequestRunners.configurableFutureRunner(10.seconds))
-    zuoraService.getAccount(Fixtures.accountNumber).map {
+    uatService.getAccount(Fixtures.accountNumber).map {
       response =>
+        logger.info(s"$response")
```

Comment:
I guess this (and below) `logger.info` statements can go from tests? Maybe in the next update?